styles.css

```diff
@@ -0,0 +1,25 @@
+body {
+    display: flex;
+    width: 100%;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
```

Comment:
this should be deleted because it won't make any changes to the content.
when you use flex-direction: column, you set the main axis to vertical. Justify-content, in this case, won't work as it works on the vertical axes as long as you don't have height specified, which is not recommended, and there is more height than the content inside it shouldn't be used.